examples/smallrng-code-gen.rs

```diff
@@ -0,0 +1,45 @@
+extern crate rand;
+use std::hint::black_box;
```

Comment:
Seems like the min MSRV is problematic here, maybe this will have to be removed.

Reply:
Either that, or...

This example serves as a non-automated test to check code generation? Can we turn it into an automated test? Otherwise I suggest it just be removed.